# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/webrtc/RTCPeerConnection-perfect-negotiation.https.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/webrtc/RTCPeerConnection-perfect-negotiation.https.html.ini
--- a/testing/web-platform/meta/webrtc/RTCPeerConnection-perfect-negotiation.https.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/webrtc/RTCPeerConnection-perfect-negotiation.https.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,25 +1,24 @@
 [RTCPeerConnection-perfect-negotiation.https.html]
   expected:
-    if (processor == "x86_64") and fission and (os == "win"): ["OK", "CRASH"]
-    if (processor == "x86") and not debug: ["OK", "TIMEOUT"]
-    if (processor == "x86") and debug: ["OK", "TIMEOUT"]
+    if (processor == "x86_64") and (os == "win") and not fission and not debug and not webrender: ["OK", "TIMEOUT"]
+    if (processor == "x86_64") and (os == "win") and fission: ["OK", "CRASH"]
+    if processor == "x86": ["OK", "TIMEOUT"]
   [Perfect negotiation setup connects with roles reversed]
     expected:
-      if (processor == "x86") and debug: ["PASS", "NOTRUN"]
-      if (processor == "x86") and not debug: ["PASS", "NOTRUN"]
+      if (processor == "x86_64") and (os == "win") and not webrender and not debug: ["PASS", "TIMEOUT"]
+      if processor == "x86": ["PASS", "NOTRUN"]
 
   [Perfect negotiation glare with roles reversed]
     expected:
-      if (processor == "x86") and debug: ["PASS", "NOTRUN"]
-      if (processor == "x86") and not debug: ["PASS", "NOTRUN"]
+      if (processor == "x86_64") and (os == "win") and not webrender and not debug: ["PASS", "NOTRUN"]
+      if processor == "x86": ["PASS", "NOTRUN"]
 
   [Perfect negotiation glare]
     expected:
-      if (processor == "x86") and debug: ["PASS", "NOTRUN"]
-      if (processor == "x86") and not debug: ["PASS", "NOTRUN"]
+      if (processor == "x86_64") and (os == "win") and not webrender and not debug: ["PASS", "NOTRUN"]
+      if processor == "x86": ["PASS", "NOTRUN"]
 
   [Perfect negotiation setup connects]
     expected:
-      if (processor == "x86") and not debug: ["PASS", "TIMEOUT"]
-      if (processor == "x86") and debug: ["PASS", "TIMEOUT"]
+      if processor == "x86": ["PASS", "TIMEOUT"]
 